spec/controllers/works/drafts_spec.rb

```diff
@@ -25,36 +26,24 @@
     end
 
     context "no user_id" do
-      it "should redirect to the user controller" do
-        get :drafts
-        expect(response).to have_http_status(:redirect)
-        expect(response).to redirect_to users_path
-      end
-
-      it "should display an appropriate error message" do
+      it "should redirect to the user controller and display an appropriate error message" do
         get :drafts
-        expect(flash[:error]).to start_with "Whose drafts did you want to look at?"
+        it_redirects_to_with_error(users_path, "Whose drafts did you want to look at?")
       end
     end
 
     context "with a valid user_id" do
       context "if the user_id requested doesn't belong to the current user" do
         it "should display an error" do
```

Comment:
This should mention the redirect as well now that it's been combined with that example